src/client/src/ui/spotTile/components/PriceMovement.tsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react'
+import styled from 'react-emotion'
```

Comment:
If you import styled from 'rt-theme' you will get full typing on the theme

- [ ]